git_machete/tests/functional/test_machete.py

```diff
@@ -1382,3 +1531,239 @@ def test_update_with_fork_point_specified(self) -> None:
+                .commit("Allow ownership links")
+                .push()
+                .new_branch("build-chain")
+                .commit("Build arbitrarily long chains")
+                .check_out("allow-ownership-link")
+                .commit("1st round of fixes")
+                .check_out("develop")
+                .commit("Other develop commit")
+                .push()
+                .new_branch("call-ws")
+                .commit("Call web service")
+                .commit("1st round of fixes")
+                .push()
+                .new_branch("drop-constraint")
+                .commit("Drop unneeded SQL constraints")
+                .check_out("call-ws")
+                .commit("2nd round of fixes")
+                .check_out("root")
+                .new_branch("master")
+                .commit("Master commit")
+                .push()
+                .new_branch("hotfix/add-trigger")
+                .commit("HOTFIX Add the trigger")
+                .push()
+                .commit_amend("HOTFIX Add the trigger (amended)")
+                .new_branch("ignore-trailing")
+                .commit("Ignore trailing data")
+                .sleep(1)
+                .commit_amend("Ignore trailing data (amended)")
+                .push()
+                .reset_to("ignore-trailing@{1}")
+                .delete_branch("root")
+                .add_remote('new_origin', 'https://github.com/user/repo.git')
+        )
+        GitAPIState.pulls = [
+            {'head': {'ref': 'ignore-trailing'}, 'user': {'login': 'github_user'}, 'base': {'ref': 'hotfix/add-trigger'}, 'number': '3', 'html_url': 'www.github.com'},
+            {'head': {'ref': 'allow-ownership-link'}, 'user': {'login': 'github_user'}, 'base': {'ref': 'develop'}, 'number': '7', 'html_url': 'www.github.com'},
+            {'head': {'ref': 'call-ws'}, 'user': {'login': 'github_user'}, 'base': {'ref': 'develop'}, 'number': '31', 'html_url': 'www.github.com'}
+        ]
+        self.launch_command("discover", "-y")
+        self.launch_command('github', 'anno-prs')
+        self.assert_command(
+            ["status"],
+            """
+            master
+            |
+            o-hotfix/add-trigger (diverged from origin)
+              |
+              o-ignore-trailing *  PR #3 (github_user) (diverged from & older than origin)
+
+            develop
+            |
+            x-allow-ownership-link  PR #7 (github_user) (ahead of origin)
+            | |
+            | x-build-chain (untracked)
+            |
+            o-call-ws  PR #31 (github_user) (ahead of origin)
+              |
+              x-drop-constraint (untracked)
+            """,
+        )
+
+    @mock.patch('git_machete.options.CommandLineOptions', FakeCommandLineOptions)
+    @mock.patch('urllib.request.urlopen', ContextManager)
+    @mock.patch('urllib.request.Request', GitAPIState)
+    def test_github_create_pr(self) -> None:
+        GitAPIState.set_initial_values()
```

Comment:
`set_initial_values()` probably belongs to the unittest's standard `setUp()` method

Reply:
A little bit outdated: since `MockGitAPIState` was extracted from `MockGitAPIRequest` i don't see a posiibility to reach it out from `setUp()` method

Reply:
Whoops right, this is no longer relevant, resolving